fishjam/media_events/server/server.proto

```diff
@@ -7,6 +7,17 @@ import "fishjam/media_events/shared.proto";
 // Defines any type of message sent from Membrane RTC Engine to Peer
 message MediaEvent {
   // SCHEMAS
+  message Track {
```

Comment:
all occurence of `map<string, string> track_id_to_metadata_json` should probably be `    map<string, Track> track_id_to_track`